src/main/java/org/fogbowcloud/saps/engine/core/dispatcher/email/keystone/KeystoneException.java

```diff
@@ -0,0 +1,14 @@
+package org.fogbowcloud.saps.engine.core.dispatcher.email.keystone;
+
+import java.io.IOException;
```

Comment:
Remove it

Reply:
Done.

Reply:
@wesleymonte 
There was confusion when I asked to remove this, I was referring to the import that was not being used:
```java
import java.io.IOException;
```
But you removed the class and used IOException/Exception, please return the class and remove only that dead import. I will add this order in the next review.

Reply:
I saw that it was solved in another way that seems ideal, it's ok